### PR TITLE
Upgrade to circleci codecov orb 1.2.3

### PR DESCRIPTION
**Pull request by kevinold**
Test latest codecov circleci orb https://github.com/codecov/codecov-circleci-orb/pull/87

### CODE REVIEW

**Comment by thomasrockhu** [comment]
@kevinold I updated the PR [here](https://github.com/cypress-io/cypress-realworld-app/pull/960)

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,12 +15,17 @@ commands:
   report-coverage:
     description: |
       Store coverage report as an artifact and send it to Codecov service.
+    parameters:
+      using_windows:
+        type: boolean
+        default: true
```
Comment (thomasrockhu) [inline .circleci/config.yml]:
```suggestion
        default: false
```

**Comment by kevinold** [comment]
@thomasrockhu Latest failure
Screen Shot 2021-05-25 at 5 06 32 PM

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: 2.1
 orbs:
   cypress: cypress-io/cypress@1.28.0
-  codecov: codecov/codecov@1.1.3
+  codecov: codecov/codecov@1.2.1
```
Comment (thomasrockhu) [inline .circleci/config.yml]:
```suggestion
  codecov: codecov/codecov@1.2.2
```
Hopefully this version will work now

**Comment by thomasrockhu** [comment]
@kevinold let me know if the new version works!

**Comment by kevinold** [comment]
@thomasrockhu Thanks, merged and we will see.

**Comment by kevinold** [comment]
Screen Shot 2021-05-26 at 1 11 41 PM
 @thomasrockhu Latest failure with v1.2.2

**Comment by thomasrockhu** [comment]
@kevinold is it possible to run the `windows` executor as `powershell.exe` as opposed to `bash.exe`? That line can just be removed

**Comment by kevinold** [comment]
@thomasrockhu Sure, we could test it.

**Comment by kevinold** [comment]
Screen Shot 2021-05-26 at 1 38 32 PM
 Latest failure from https://github.com/cypress-io/cypress-realworld-app/pull/956/commits/92324eee04695d4e606d65fe7e56225be718ade5.  Seems we need `bash.exe`

File: .circleci/config.yml [hunk 3]
```diff
@@ -15,12 +15,17 @@ commands:
   report-coverage:
     description: |
       Store coverage report as an artifact and send it to Codecov service.
+    parameters:
+      using_windows:
+        type: boolean
+        default: false
```
Comment (thomasrockhu) [inline .circleci/config.yml]:
```suggestion
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -15,12 +15,17 @@ commands:
...
     steps:
       - store_artifacts:
           path: coverage
       - run: npx nyc report --reporter=text || true
       - codecov/upload:
           file: coverage/coverage-final.json
+          using_windows: << parameters.using_windows >>
```
Comment (thomasrockhu) [inline .circleci/config.yml]:
```suggestion
```

File: .circleci/config.yml [hunk 5]
```diff
@@ -155,7 +160,7 @@ windows-workflow: &windows-workflow
         executor:
           # executor comes from the "windows" orb
           name: win/default
-          shell: bash.exe
+          shell: powershell.exe
```
Comment (thomasrockhu) [inline .circleci/config.yml]:
```suggestion
          shell: bash.exe
```

File: .circleci/config.yml [hunk 6]
```diff
@@ -184,7 +189,7 @@ windows-workflow: &windows-workflow
...
```
Comment (thomasrockhu) [inline .circleci/config.yml]:
```suggestion
          shell: bash.exe
```

File: .circleci/config.yml [hunk 7]
```diff
@@ -196,7 +201,8 @@ windows-workflow: &windows-workflow
         requires:
           - Setup Windows
         post-steps:
-          - report-coverage
+          - report-coverage:
+              using_windows: true
```
Comment (thomasrockhu) [inline .circleci/config.yml]:
```suggestion
          - report-coverage
```

File: .circleci/config.yml [hunk 8]
```diff
@@ -1,7 +1,7 @@
 version: 2.1
 orbs:
   cypress: cypress-io/cypress@1.28.0
-  codecov: codecov/codecov@1.1.3
+  codecov: codecov/codecov@1.2.2
```
Comment (thomasrockhu) [inline .circleci/config.yml]:
```suggestion
  codecov: codecov/codecov@1.2.3
```

**Review by thomasrockhu — COMMENTED** [review]
@kevinold Made some fixes to `1.2.3`

**Comment by kevinold** [comment]
@thomasrockhu 
Screen Shot 2021-05-27 at 12 23 18 PM

**Comment by thomasrockhu** [comment]
@kevinold sorry about that, it should be up now

**Comment by kevinold** [comment]
@thomasrockhu 1.2.3 solved the issue!  Thanks!

**Comment by thomasrockhu** [comment]
@kevinold woo hoo! thanks for your patience here.

**Pull request merged**